### DISCUSSION TITLE
May I ask if it is possible to use C/C++for secondary development? I would like to use the HTTP service inside to do some other business.

### DISCUSSION THREAD

**Discussion by yanzhang920817** [rauc/rauc#1587] [Q&A]
I downloaded the `v1.12` version of `RAUC`, but I noticed that the files inside are all. go. Can I use C/C++for secondary development? I want to use the HTTP service inside to do some other business.

**Comment by jluebbe** [comment]
What do you mean? RAUC is written in C: https://github.com/rauc/rauc/tree/master/src

RAUC has a D-Bus API, so it can be easily managed from other processes, such as a web server.
